wiki/History_of_osu!/2016/en.md

```diff
@@ -2,125 +2,181 @@
+- [Fleshgod Apocalypse - The Deceit / The Violation (Mazzerin)](https://osu.ppy.sh/beatmapsets/367631)
+- [sHimaU - Slotcore is Dead (Zapy)](https://osu.ppy.sh/beatmapsets/37065)
+- [DJ Mass MAD Izm\* - SHAKUNETSU Beach Side Bunny (rustbell)](https://osu.ppy.sh/beatmapsets/409345)
+- [Blue Stahli - Anti You (thatguy0815)](https://osu.ppy.sh/beatmapsets/42567)
+- [Cardboard Box - The Limit Does Not Exist (Blue Dragon)](https://osu.ppy.sh/beatmapsets/429469)
+- [Seether - Fake It ([Dellirium])](https://osu.ppy.sh/beatmapsets/48738)
+- [Vektor - Recharging the Void (Mazzerin)](https://osu.ppy.sh/beatmapsets/489236)
+
+Despite the first additions, users were reminded that the new ranking status was still in "open testing", and that the team still was not sure of how the specifics of getting a beatmap into Loved should work. Many different ideas for this were thrown around, although the first to be tested was a system of weekly community voting from a pool of ten maps chosen based on a "variety of factors" [[28,29]][r].
+
+---
+
+## References
+
+1. [osu!weekly #47](https://osu.ppy.sh/home/news/2016-02-09-osuweekly-47)
+2. [28 January 2016 changelog](https://osu.ppy.sh/home/changelog/cuttingedge/20160128)
+3. [Mapset and Spread Restructure (Proposed)](https://osu.ppy.sh/community/forums/topics/420223)
+4. [MEET EPHEMERAL!](https://blog.ppy.sh/post/139478794378/meet-ephemeral "ppy blog")
+5. [Timeline of Events Surrounding Recent Changes](https://docs.google.com/document/d/1VlFUIte8ho4tssRCucSBt96nTVAhsCLfJZcFVgzHjuk/edit "Google Docs")
+6. [osu!weekly #48](https://osu.ppy.sh/home/news/2016-02-16-osuweekly-48)
+7. [osu!go not worth it?](https://osu.ppy.sh/community/forums/topics/425108)
+8. [2016-02 MEETING NOTES](https://blog.ppy.sh/post/140259300353/2016-02-meeting-notes "ppy blog")
+9. [Modding v2: Modding Panel - Feedback Thread](https://osu.ppy.sh/community/forums/topics/442285)
+10. [osu!weekly #57](https://osu.ppy.sh/home/news/2016-04-20-osuweekly-57)
+11. [15 April 2016 changelog](https://osu.ppy.sh/home/changelog/cuttingedge/20160415)
+12. [osu! | How medals looks like? (Animations) | Achievements 20/04/2016](https://www.youtube.com/watch?v=sdGe-Gv38Yw "YouTube")
+13. [Meet Maria - osu!mania's new mascot!](https://osu.ppy.sh/home/news/2016-04-20-meet-maria-osumanias-new-mascot)
+14. [Changes to the Quality Assurance Team](https://osu.ppy.sh/community/forums/topics/447417)
+15. [osu! "circles!" Remix Contest](https://osu.ppy.sh/home/news/2016-05-06-osu-circles-remix-contest)
+16. [EARLY-BIRD CONTRIBUTORS](https://blog.ppy.sh/post/144539340703/early-bird-contributors "ppy blog")
+17. [31 May 2016 (18:22 UTC) Tweet](https://twitter.com/ppy/status/737529593184149504 "Twitter")
+18. [Mapping With Rewards #1 - Bounties Complete!](https://osu.ppy.sh/home/news/2016-06-01-mapping-with-rewards-1-bounties-complete)'
+19. [9 June 2016 (3:18 PM UTC)) Tweet](https://twitter.com/ppy/status/741107246965235713 "Twitter")
```

Comment:
these have been removed since https://github.com/ppy/osu-wiki/pull/6060

```suggestion
19. [9 June 2016 (3:18 PM UTC)) Tweet](https://twitter.com/ppy/status/741107246965235713)
```